homeassistant/components/xiaomi_miio/fan.py

```diff
@@ -690,6 +747,10 @@ def __init__(self, name, device, model, unique_id):
             self._device_features = FEATURE_FLAGS_AIRPURIFIER_2S
             self._available_attributes = AVAILABLE_ATTRIBUTES_AIRPURIFIER_2S
             self._speed_list = OPERATION_MODES_AIRPURIFIER_2S
+        elif self._model == MODEL_AIRPURIFIER_3 or self._model == MODEL_AIRPURIFIER_3H:
```

Comment:
I was rechecking this PR and saw these and wondered why the one below is not merged here, before I realized that one is `_3` and one `_V3`. Do you think it would make sense to add `MIOT` somewhere in there to make it easier to differentiate?

Reply:
I didn't ever see V3 version but it's something older and different from 3/3h from 2019...
I don't think adding MIOT would be the best solution. I think in not so distant future we'll need to have 2/2s/Pro support with MiOT protocol...

Reply:
Or simply use some part of the real model number? It's mildly confusing to differentiate between those constants. Anyway, this is not a blocking issue.

Reply:
One last thing to do before merging, please update the documentation (for the model listing & the new service): https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/fan.xiaomi_miio.markdown

Reply:
Made one

Reply:
Thanks, let's get this merged, the docs will follow as soon as someone reviews it! Thanks for your hard work and patience on this! :1st_place_medal: